.github/workflows/actions.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+
+  push:
+    branches: 
+      - '**'
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Clone and install circom
+        uses: actions/checkout@v3
+        with:
+          repository: iden3/circom
+          path: circom
+          ref: master
+      - name: Install latest rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Cache build artifacts
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/target
+            ~/.cargo
+          key: aes-gcm-siv-0.0.1  
+      - run: cargo build --release --manifest-path=circom/Cargo.toml && cargo install --path circom/circom
```

Comment:
Can't we directly do cargo install circom instead of cloning and using that?

Reply:
I am not sure, I did it the same way as it was in circom-ed25519 repo.

Reply:
I think we can give it a try just do cargo install circom.

Reply:
`error: could not find 'circom' in registry 'crates-io' with version '*'`
`circom` is not listed on crate.io